cvat-ui/src/containers/annotation-page/standard-workspace/objects-side-bar/object-buttons.tsx

```diff
@@ -29,11 +29,13 @@ interface StateToProps {
     outsideDisabled: boolean;
     hiddenDisabled: boolean;
     keyframeDisabled: boolean;
+    editedState: EditingState,
```

Comment:
As it comes from global store and we anyway use `mapStateToProps` here, I do not see the reason to pass as own props

Reply:
We dont pass new value as own prop, Im not sure what you mean here